webclient/pas/migrations/0004_triggers.py

```diff
@@ -7,7 +7,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ("core", "0005_triggers"),
+        ("core", "0009_ident_sequences_v2"),
```

Comment:
Ahoj Honzo, prosím tě, má to být takto - tj. že ty migrace se nespouštějí v pořadí dle jejich čísel? Asi to vyloženě nevadí, ale raději ověřuji, zda to není chyba.

Reply:
tohle je migrace pas, ktera musi cekat na migraci core. zmatli te asi cisla.

Reply:
Jo, máš pravdu, díky za upozornění.